test/testcafe/spec/DeviceChallengePollViewLoopbackFailfast_spec.js

```diff
@@ -0,0 +1,68 @@
+import { RequestLogger, RequestMock } from 'testcafe';
+import DeviceChallengePollPageObject from '../framework/page-objects/DeviceChallengePollPageObject';
+import IdentityPageObject from '../framework/page-objects/IdentityPageObject';
+import identifyWithDeviceProbingLoopback from '../../../playground/mocks/idp/idx/data/identify-with-device-probing-loopback';
+import loopbackChallengeNotReceived from '../../../playground/mocks/idp/idx/data/identify-with-device-probing-loopback-challenge-not-received';
+
+let failureCount = 0;
+
+const logger = RequestLogger(/introspect|probe|challenge/, { logRequestBody: true, stringifyRequestBody: true });
+
+const mock = RequestMock()
+  .onRequestTo('http://localhost:3000/idp/idx/introspect')
+  .respond(identifyWithDeviceProbingLoopback)
+  .onRequestTo('http://localhost:3000/idp/idx/authenticators/poll')
+  .respond((req, res) => {
+      res.statusCode = '200';
+      if (failureCount === 2) {
+          res.setBody(loopbackChallengeNotReceived);
+      } else {
+          res.setBody(identifyWithDeviceProbingLoopback);
+      }
+  })
+  .onRequestTo('http://localhost:2000/probe')
+  .respond(null, 500, { 'access-control-allow-origin': '*' })
+  .onRequestTo('http://localhost:6511/probe')
+  .respond(null, 500, { 'access-control-allow-origin': '*' })
+  .onRequestTo('http://localhost:6512/probe')
```

Comment:
For failfast approach, all the authenticator request should respond with error.

Reply:
Sorry. It's a copy and paste error. I copied the wrong file after fork the repo.